### PR TITLE
Port scripts to support both Python 2 and 3

### PR DESCRIPTION
**Pull request by chuan-wang**
Associated with validation 21_11

### CODE REVIEW

**Comment by chuan-wang** [comment]
Validation 21_11 has been approved. /CW

File: setup.py [hunk 1]
```diff
@@ -7,11 +7,11 @@
     install_requires = []
 
 setup(name = "statusdb",
-    version = "1.0",
+    version = "1.0.0",
```
Comment (aanil) [inline setup.py:10]: Should we increment the version number?

Comment (chuan-wang) [reply setup.py:10]: Hi, in `__init__.py` the version number was "0.2.0". And I guess it is OK to have the new version at "1.0.0"?

Comment (aanil) [reply setup.py:10]: Ok, if you think its fine ☺️

Comment (chuan-wang) [reply setup.py:10]: 👍

**Review by aanil — APPROVED** [review]
👍 Looks ok to me!

**Pull request merged**